changelogs/fragments/58115_nmcli.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- nmcli - Typecast parameters to string as required (https://github.com/ansible/ansible/issues/59095).
```

```suggestion
- nmcli - typecast parameters to string as required (https://github.com/ansible/ansible/issues/59095).
```
https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to
it's been merged recently.